app/Http/Controllers/ViewAssetsController.php

```diff
@@ -96,17 +97,12 @@ public function getRequestItem(Request $request, $itemType, $itemId = null, $can
 
         $user = auth()->user();
 
-        $logaction = new Actionlog();
-        $logaction->item_id = $data['asset_id'] = $item->id;
-        $logaction->item_type = $fullItemType;
-        $logaction->created_at = $data['requested_date'] = date('Y-m-d H:i:s');
-
+        $data['requested_date'] = date('Y-m-d H:i:s'); //FIXME - shouldn't this be recorded somewhere?
         if ($user->location_id) {
-            $logaction->location_id = $user->location_id;
+            $item->setLocation($user->location);
```

Comment:
What am I trying to do here?! There *is* no `setLocation` - I think I must've meant `setLogLocationOverride()` - because there isn't a `setLocation()` method anywhere. I'll fix.